eos_data_distribution/producers/usb.py

```diff
@@ -107,7 +107,6 @@ def main():
     GLib.unix_signal_add(GLib.PRIORITY_DEFAULT, signal.SIGTERM, signal_cb)
     monitor = Gio.VolumeMonitor.get()
     store = SimpleStoreProducer(prefix=SUBSCRIPTIONS_SOMA,
-                                split=defaults.ENDLESS_NDN_CACHE_PATH,
```

Comment:
so you are going to need to do at least that for usb (or you'll break it) we can use base instead of split and deprecate split.